plugin/httptrace/clienttrace.go

```diff
@@ -45,112 +44,124 @@ type clientTracer struct {
 
 	tr trace.Tracer
 
-	levels map[string]trace.Span
-	root   trace.Span
-	mtx    sync.Mutex
+	activeHooks map[string]trace.Span
+	root        trace.Span
+	mtx         sync.Mutex
 }
 
 func newClientTracer(ctx context.Context) *clientTracer {
 	ct := &clientTracer{
-		Context: ctx,
-		levels:  make(map[string]trace.Span),
+		Context:     ctx,
+		activeHooks: make(map[string]trace.Span),
 	}
 	ct.tr = global.TraceProvider().GetTracer("go.opentelemetry.io/otel/plugin/httptrace")
-	ct.open("http.request")
+	ct.start("http.request", "http.request")
 	return ct
 }
 
-func (ct *clientTracer) open(name string, attrs ...core.KeyValue) {
-	_, sp := ct.tr.Start(ct.Context, name, trace.WithAttributes(attrs...), trace.WithSpanKind(trace.SpanKindClient))
+func (ct *clientTracer) start(hook, span string, attrs ...core.KeyValue) {
```

Comment:
`span` feels like it could be confusing given a lot of folks might name the result of `tr.Start` `span` rather than `sp`. Consider using `spanName` or similar if you don't like `name`?

Reply:
Changed to `spanName` to be a little more clear  :)